pkg/pipeline/pipeline.go

```diff
@@ -115,7 +115,9 @@ func (p *Pipeline) SendToFirstNode(msg *Message) error {
 	select {
 	case p.header <- msg:
 	default:
-		return cerror.ErrPipelineTryAgain.GenWithStackByArgs()
+		// Do not call `GenWithStackByArgs` in the hot path,
+		// it consumes lots of CPU.
+		return cerror.ErrPipelineTryAgain
```

Comment:
What about creating a global error using `cerror.ErrPipelineTryAgain.FastGenByArgs()` and returning with it

Reply:
Do we have to call `GenXXX` or `FastGenXXX`? Is there any error handling idiom of TiCDC?

Reply:
`cerror.ErrPipelineTryAgain` is a global error instance, which is used to generate an error. To use a global rfc error is dangerous because we may change it by accident.